fdm-app/app/routes/farm.create.$b_id_farm.fields.$b_id.tsx

```diff
@@ -73,9 +73,11 @@ const FormSchema = z.object({
         .min(3, {
             message: "Naam van perceel moet minimaal 3 karakters bevatten",
         }),
-    b_area: z.coerce.number({
-        required_error: "Oppervlakte van perceel is verplicht",
-    }),
+    b_area: z.coerce
+        .number({
+            required_error: "Oppervlakte van perceel is verplicht",
+        })
+        .optional(),
     b_lu_catalogue: z.string({
```

Comment:
if your statement is "area is mandatory", what implies .optional() here?

Reply:
Will be resolved in e40fd2b87a3380c7f3634b75639e944085f611bd